libs/experimental/langchain_experimental/graph_transformers/llm.py

```diff
@@ -770,7 +771,7 @@ def convert_to_graph_documents(
         Returns:
             Sequence[GraphDocument]: The transformed documents as graphs.
         """
-        return [self.process_response(document) for document in documents]
+        return [self.process_response(document, config) for document in documents]
 
     async def aprocess_response(self, document: Document) -> GraphDocument:
```

Comment:
could we make similar changes on the async equivalents?

Reply:
Done, thank you!